ui/components/multichain/toast/toast.tsx

```diff
@@ -66,16 +81,36 @@ export const Toast = ({
     <BannerBase
       data-theme={theme === ThemeType.light ? ThemeType.dark : ThemeType.light}
       onClose={onClose}
-      borderRadius={borderRadius}
+      backgroundColor={BackgroundColor.backgroundSection}
+      borderWidth={1}
+      borderColor={BorderColor.borderMuted}
+      borderRadius={borderRadius || BorderRadius.XL}
       data-testid={dataTestId ? `${dataTestId}-banner-base` : undefined}
       className={`toasts-container__banner-base ${className}`}
     >
-      <Box display={Display.Flex} gap={4} data-testid={dataTestId}>
+      <Box
+        display={Display.Flex}
+        gap={3}
+        data-testid={dataTestId}
+        {...contentProps}
+      >
         {startAdornment}
         <Box>
-          <Text className="toast-text" variant={textVariant}>
+          <Text
+            className="toast-text"
+            variant={textVariant || TextVariant.bodyMdMedium}
+          >
             {text}
           </Text>
+          {description && (
+            <Text
+              className="toast-text"
+              variant={descriptionVariant || TextVariant.bodySm}
+              color={TextColor.textAlternativeSoft}
```

Comment:
This color should be deprecated sorry! Could we change it to `TextAlternative`

```suggestion
              color={TextColor.textAlternative}
```

Reply:
oh.. I updated it thank you.